docs/modules/ROOT/pages/how-to/use-github-actions.adoc

```diff
@@ -0,0 +1,228 @@
+= How to use GitHub Actions with {product}
+
+image:logos/github-actions.png[role="related thumb right",alt="GitHub Actions logo",width=120,height=120] https://docs.github.com/en/actions[GitHub Actions] enables developers to automate, customize, and execute your software development workflows right in your repository.
+
+This page will describe the required steps to use GitHub Actions successfully with {product}.
+
+IMPORTANT: This information is provided as a courtesy for {product} users. We don't provide support for GitHub Actions, and in case of issues, you are invited to check the https://docs.github.com/en/actions[official documentation].
+
+== Requirements
+
+To follow this guide, please make sure that you have the following elements:
+
+GitHub account:: You can create one for free at https://github.com.
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+== Procedure
+
+This section explains the steps required to use GitHub Actions with {product}.
+
+=== 1. Login to {product}
+
+include::partial$login-in-terminal.adoc[]
+
+. Create a new project called "[YOUR_USERNAME]-fortune-go"
++
+[source,shell]
+--
+oc new-project [YOUR_USERNAME]-fortune-go
+--
+
+=== 2. Clone the sample "Fortune in Go" project
+
+The https://gitlab.com/vshn/applications/fortune-go["Fortune in Go"] project provides a very simple Go application to start with.
+
+. Login to your https://github.com[GitHub] account.
+. Click on the `+` sign on the top right and select "Import repository" as shown in the screenshot below:
++
+image::how-to/github-import.png[]
+
+. Enter the URL of the project:
+** `https://gitlab.com/vshn/applications/fortune-go`
+
+. Enter `fortune-go` as the name of the project in your profile.
+
+. Clone the project:
++
+[source,shell]
+--
+git clone git@github.com:[YOUR_GITHUB_USERNAME]/fortune-go.git
+--
+
+. `cd` into it:
++
+[source,shell]
+--
+cd fortune-go
+--
+
+. Since the code has been imported from another repository, GitHub Actions aren't automatically available. Enable them by going to the menu:Settings[Actions > General] screen and selecting the "Allow all actions" option.
+
+=== 3. Add a Deployment File
+
+Add a file named `deployment.yaml` at the root of the project with the following information:
+
+[source,yaml]
+--
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
Does this require a feature provided only by `DeploymentConfig`? If not, please consider to use `Deployment` which is more universal in the Kubernetes world.

Reply:
I'm using `run: oc rollout latest dc/fortune-go` at the end of the workflow, so that the latest version of the container goes live, and this requires `DeploymentConfig` afaik (maybe I'm mistaken? But I tried this with Deployment without success)

Reply:
If there is a different way to ensure that the latest version of the container goes live without having to `oc delete` first, and which uses plain `Deployment`, I'm all ears :)